docs/sources/enterprise/access-control/about-rbac.md

```diff
@@ -38,17 +37,27 @@ Grafana includes the following basic roles:
 - Editor
 - Viewer
 
-Each basic role is comprised of a number of _fixed roles_ that control the permissions a basic role grants. For example, the viewer basic role contains the following fixed roles among others:
+Each basic role is comprised of a number of _permissions_. For example, the viewer basic role contains the following permissions among others:
 
-- `fixed:datasources:id:reader`: Enables the viewer to see the ID of a data source.
-- `fixed:organization:reader`: Enables the viewer to see a list of organizations.
-- `fixed:annotations:reader`: Enables the viewer to see annotations that other users have added to a dashboard.
-- `fixed:annotations.dashboard:writer`: Enables the viewer to add annotations to a dashboard.
+- `Action: datasources.id:read, Scope: datasources:*`: Enables the viewer to see the ID of a data source.
+- `Action: orgs:read`: Enables the viewer to see their organization details
+- `Action: annotations:read, Scope: annotations:*`: Enables the viewer to see annotations that other users have added to a dashboard.
+- `Action: annotations:create, Scope: annotations:type:dashboard`: Enables the viewer to add annotations to a dashboard.
+- `Action: annotations:write, Scope: annotations:type:dashboard`: Enables the viewer to modify annotations of a dashboard.
+- `Action: annotations:delete, Scope: annotations:type:dashboard`: Enables the viewer to remove annotations from a dashboard.
 
-You can use RBAC to modify the fixed roles associated with any basic role, to modify what viewers, editors, or admins can do. For more information about the fixed roles associated with each basic role, refer to [Basic role definitions]({{< relref "./rbac-fixed-basic-role-definitions#basic-role-assignments" >}}).
+You can use RBAC to modify the permissions associated with any basic role, which changes what viewers, editors, or admins can do. For more information about the permissions associated with each basic role, refer to [Basic role definitions]({{< relref "./rbac-fixed-basic-role-definitions#basic-role-assignments" >}}).
 
 > **Note:** You must assign each Grafana user a basic role.
 
+To interact with the API and view or modify basic roles permissions, refer to the following table that maps basic role names to the associated UID:
```

Comment:
This information seems to be included already in "List permissions associated with roles" section under Manage RBAC roles. What do you think about simply referencing it to that page? For example

"To see the UIDs associated with each basic role, refer to Manage RBAC roles"